src/main/webapp/admin_dashboard.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Admin Dashboard</title>
+		<link rel="stylesheet" href="admin_dashboard_style.css">
+		<script src='https://kit.fontawesome.com/a076d05399.js'></script>
+        <script src="https://www.gstatic.com/charts/loader.js"></script>
+        <script src="admin_dashboard_script.js"></script>
+	</head>    
+	<body onresize="drawChart()">
+		<div class="wrapper">
+			<nav>
+				<header>NeighborGood</header>
+				<ul>
+					<li><a class="active">Dashboard</a></li>
```

Comment:
This looks good! You might want to think though what you want these to do when clicked and add some 'fake functionality' for the prototype :)

Reply:
Agreed! Looks great, and it would be nice to see what you intend these sections to do. (That should probably be a follow-up PR, though.)